### ISSUE TITLE
HTML composer is pretty unintuitive

### ISSUE THREAD

**Issue by gabrc52** [roundcube/roundcubemail#7986]
I've used Roundcube for many days without realizing I could compose HTML (more than plain text) email.

For some reason, plain text is the default. I think it'd make more sense to keep HTML mail as default.

But the reason I didn't realize you couldn't compose HTML email is that the icon is that of a picture, and I only clicked it when I wanted to attach a picture

![image](https://user-images.githubusercontent.com/24363938/113353935-a3f32680-92fb-11eb-95c0-f7a03cb0f5f7.png)

I'm not sure what the better icon could be, but here are some ideas:

**B** _I_ 𝚞̲

![image](https://user-images.githubusercontent.com/24363938/113354260-254ab900-92fc-11eb-8383-863424d7543d.png)

Or just a text button that says "Use formatting".

**Comment by thomascube** [comment]
I agree and I like the suggestion with the underlined __A__. This also the icon Gmail uses to enable text formatting.

**Comment by alecpl** [comment]
I'm not sure I like the suggested ones, but I'm open to discussion.

There's bold, underline, strike-through, italic icons in Font-Awesome, but I didn't find anything like this one.

Maybe we could borrow from TinyMCE: https://github.com/tinymce/tinymce/blob/develop/modules/oxide-icons-default/src/svg/text-color.svg or create something custom with https://fontawesome.com/icons/font?style=solid

I don't know. Maybe adding a label to the button would be the better solution.
